platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java

```diff
@@ -374,6 +378,38 @@ public void setBundleLocation(String pid, String location) throws IOException {
         config.setBundleLocation(location);
     }
 
+    public boolean updateGuestClaimsProfile(String pid, Map<String, Object> configurationTable)
+            throws IOException {
+        try {
+            Object profileObj = configurationTable.get(PROFILE_KEY);
+            if (profileObj == null) {
+                return false;
+            }
+
+            if (!(profileObj instanceof String)) {
+                LOGGER.debug("Selected guest claims profile was not a String");
+                return false;
+            }
+
+            String profile = (String) profileObj;
+            guestClaimsHandlerExt.setSelectedClaimsProfileName(profile);
+
+            Map<String, Object> currentUiConfig = getProperties(UI_CONFIG_PID);
+            Map<String, Object> defaultBannerMarkings =
+                    guestClaimsHandlerExt.getSelectedClaimsProfileBannerConfigs();
```

Comment:
The default banner markings are not inferred from system high or low; they exist as configurations linked to the install profiles for security. Any project extending DDF can provide them in `~/etc/ws-security/profile-banners` as needed.